charts/zitadel/Chart.yaml

```
@@ -2,8 +2,8 @@ apiVersion: v2
 name: zitadel
 description: A Helm chart for ZITADEL
 type: application
-appVersion: "v2.51.3"
-version: 7.15.0
+appVersion: "v2.54.1"
```

We had a bug in v2.54.1 that probably also explains that the tests fail. 

```suggestion
appVersion: "v2.54.2"
```